Remove CSS classes from Vet Review

labkey-martyp opened the pull request:
#### Rationale
CSS classes were added in an inline scripting refactor. These classes are on the core EHR vet review but that styling is not wanted here.

#### Changes
* Remove CSS classes

labkey-martyp committed:
remove classes

jryurkanin commented:
@labkey-martyp @ankurjuneja  Was it necessary to also get rid of the vrdc-row class? Brent noticed some undesired effects of this fb as it currently is. It removes the behavior of opening a Clinical Remarks pop-up, and Brent thinks it's related to removing the vrdc-row class here.

labkey-martyp committed:
Add vrdc-row back

labkey-martyp commented:
> @labkey-martyp @ankurjuneja Was it necessary to also get rid of the vrdc-row class? Brent noticed some undesired effects of this fb as it currently is. It removes the behavior of opening a Clinical Remarks pop-up, and Brent thinks it's related to removing the vrdc-row class here.

Apologies, I just removed the css classes that were added, I didn't see that the vrdc-row was being used for the handler. I added it back.
